src/Event.php

```diff
@@ -178,6 +187,30 @@ public function setSdkIdentifier(string $sdkIdentifier): void
         $this->sdkIdentifier = $sdkIdentifier;
     }
 
+    /**
+     * Gets the version of the SDK package that generated this Event.
+     *
+     * @return string
```

Comment:
```suggestion
     * @return string
     *
     * @internal
```
Let's make these functions internal.